go-selinux/label/label_linux.go

```diff
@@ -18,7 +18,7 @@ var validOptions = map[string]bool{
 	"level":    true,
 }
 
-var ErrIncompatibleLabel = errors.New("Bad SELinux option z and Z can not be used together")
+var ErrIncompatibleLabel = errors.New("bad SELinux option: z and Z can not be used together")
```

Comment:
👍  Did a quick search to see if there's any code string-matching the error, but I don't think there is (nothing found outside of `vendor/` AFAICS https://grep.app/search?case=true&q=Bad+SELinux+option

Reply:
Actually; it looks like nobody may be using the `ErrIncompatibleLabel` error as a sentinel error

https://grep.app/search?case=true&q=.ErrIncompatibleLabel


I don't know if there would be any reason for anyone to treat this error special; if there's not, we could consider deprecating it.

(but that's definitely unrelated to this PR!)